src/js/themes/base.d.ts

```diff
@@ -951,6 +952,64 @@ export interface ThemeType {
   };
   pagination?: {
     container?: BoxProps;
+    button?: {
+      color?: ColorType;
+      active?: {
+        background?: {
+          color?: ColorType;
+        };
+      };
+      hover?: {
+        background?: {
+          color?: ColorType;
+        };
+        color?: ColorType;
+      };
+    };
+    control?: {
+      extend?: ExtendType;
+      size?: {
+        small?: {
+          border?: {
+            radius?: string;
+            width?: string;
+          };
+          font?: {
+            size?: string;
+            height?: string;
+            maxWidth?: string;
+          };
+          height?: string;
+          width?: string;
+        };
+        medium?: {
+          border?: {
+            radius?: string;
+            width?: string;
+          };
+          font?: {
+            size?: string;
+            height?: string;
+            maxWidth?: string;
```

Comment:
What would this value be and how does it affect styling?

Reply:
Good catch. Not needed here.

Reply:
Removed.